### ISSUE TITLE
Thoughts about cross-country usage?

### ISSUE THREAD

**Issue by HrFlorianHoffmann** [corona-warn-app/cwa-documentation#174]
While appreciating the effort that goes into this national app I am still somewhat puzzled how each European country now comes up with its own individual solution. Did we think about cross-country usage? From the architecture comments I deduce that data exchange with other apps and countries is neither built in nor envisioned for the future. What does that mean for the everyday use cases?

For example, would we recommend foreign users who enter Germany to install the app? If so, the app’s warning of a possible Corona contact will reach that user, but not his social network, which might extend into other countries and apps. Should the app thus advise the user to manually trigger his other apps in case he tests positive?

**Comment by treysis** [comment]
https://github.com/corona-warn-app/cwa-documentation/issues/55

**Comment by LukasMasuch** [comment]
Thanks for the question. We already work on cross-country interoperability with the goal to provide a seamless experience for travellers. Users visiting other interoperable countries will not need to install additional national apps for most scenarios. I added some more information in my comment here: https://github.com/corona-warn-app/cwa-documentation/issues/55#issuecomment-636529920. Closing this as a duplicate #55.
